src/test/java/com/hillel/items_exchange/security/SecurityConfigIntegrationTest.java

```diff
@@ -151,11 +202,13 @@ private String obtainToken(UserLoginDto loginDto) throws Exception {
     }
 
     private void createNonExistAdvertisementDto() {
```

Comment:
Someday we will extract this logic to a util class, but not today... :smile:

Reply:
This method has too many fields inside. So... not today... 😄

Reply:
I saw how well solved similar problem Jack11M in his PR, and i will add solution in next commit